### DISCUSSION TITLE
想让Picker 里面的背景色变为其他自定义的颜色 尝试发现无效

### DISCUSSION THREAD

**Discussion by wencai123** [ant-design/ant-design-mobile#5171] [Q&A]
<img width="406" alt="image" src="https://user-images.githubusercontent.com/82213523/167412510-a7fbfdb1-0265-4989-a046-40abc81f3ae3.png">

**Comment by awmleer** [comment]
你是想自定义成暗黑模式么，目前的确是还不支持，只能先强制覆盖一下样式了
